recipe/symfony-flex.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Deployer;
+
+require_once __DIR__ . '/symfony3.php';
+
+/**
+ * Symfony Flex Configuration
+ */
+
+//No need to clear anything
+set('clear_paths', []);
+
+// Symfony shared dirs
+set('shared_dirs', ['var/log', 'var/sessions']);
```

Comment:
same here

Reply:
Also nope. And 'var/logs', not 'var/log'

Reply:
The default directory for logs is `var/log`  (since 3.4)